tests/core/full_node/test_block_height_map.py

```diff
@@ -34,7 +34,7 @@ async def new_block(
 ):
     if db.db_version == 2:
         cursor = await db.db.execute(
-            "INSERT INTO block_records VALUES(?, ?, ?, ?)",
+            "INSERT INTO full_blocks VALUES(?, ?, ?, ?)",
```

Comment:
Should these tests not represent the full real db?  Should they not use the actual schema creation code?

Reply:
well, I kept this test as small as possible, by only building a table with columns this class actually use. I don't think there's much value in making the test add more fields here, and come up with fake data, when nobody reads them.
Specifically, generating fake blocks can be quite expensive, even if you just fake all hashes with random bytes